.code-samples.meilisearch.yaml

```diff
@@ -76,7 +81,13 @@ reset_pagination_settings_1: |-
 get_faceting_settings_1: |-
   index('books').faceting
 update_faceting_settings_1: |-
-  index('books').update_faceting({ max_values_per_facet: 2 })
+  index('books').update_faceting({
+    max_values_per_facet: 2,
+    sort_facet_values_by: {
```

Comment:
There is an issue open of this, can you confirm this code sample is still valid @brunoocasali?